src/components/banner/index.tsx

```diff
@@ -68,18 +68,24 @@ export const Banner = (props: IBannerProps) => {
     fontSize: '16px',
     marginBottom: '16px'
   };
-  const bannerButtons = {
-    marginTop: '20px'
+  const bannerButtons: CSSProperties = {
+    marginTop: '20px',
+    flex: '1',
+    flexDirection: 'row',
+    display: 'flex'
   };
   const defaultButtonStyles: CSSProperties = {
+    maxWidth: 'calc(50% - 32px)',
```

Comment:
why 32px? would prefer some percentage here instead so it's more responsive unless this needs to be a fixed pixel value.

Reply:
We've added a margin for the button, so using a percentage might not function correctly in this context.